codegen/smithy-aws-ruby-codegen/src/main/java/software/amazon/smithy/aws/ruby/codegen/Sigv4Auth.java

```diff
@@ -115,7 +115,6 @@ metadata service (IMDS) on an EC2 instance.
                 })
                 .klass("AWS::SDK::Core::Middleware::SignatureV4")
                 .step(MiddlewareStackStep.FINALIZE)
-                .addConfig(signer)
```

Comment:
Why is signer removed here?

Reply:
It was being added twice to the builder.